### ISSUE TITLE
Share what to do if you already have an ssh key

### ISSUE THREAD

**Issue by maneesha** [swcarpentry/git-novice#954]
### How could the content be improved?

In the [ssh setup section](https://swcarpentry.github.io/git-novice/aio.html#ssh-background-and-setup), we show how to check if you already have an ssh key set up, and what to do if you don't.  It would also be useful to share what to do if you do already have it set up.  Perhaps something along the lines of "If you already have your ssh key set up, skip down to step XX."

### Which part of the content does your suggestion apply to?

https://swcarpentry.github.io/git-novice/aio.html#ssh-background-and-setup

**Comment by narmadanagar** [comment]
I agree, user can Enter `ls -al ~/.ssh` to see if existing SSH keys are present.

`$ ls -al ~/.ssh`
`# Lists the files in your .ssh directory, if they exist`

**Comment by jas58** [comment]
Does XX mean 3.2?  because the last output bash box lists the type of files in .ssh directory to expect, right?

**Comment by erinmgraham** [comment]
Hi All, 

Thank you for your thoughtful comments on this episode. I am leaning towards removing the check for other keys - see my comments on #950; however I do acknowledge there are pros and cons to any solution we choose.

Given the issues opened around the existing ssh content and other issues opened to replace ssh with it with e.g., https/gcm/pat, I will  label this as a discussion and speak with the other maintainers to work out a way forward.
